app/app/sampledata/com.example.trackflix.database.TrackableDatabase/1.json

```diff
@@ -0,0 +1,58 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "d4cb6ab8b9b3f53a03d076973a703861",
+    "entities": [
+      {
+        "tableName": "trackable_table",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `title` TEXT NOT NULL, `currentProgress` INTEGER NOT NULL, `goal` INTEGER NOT NULL, `type` TEXT NOT NULL)",
```

Comment:
In future, some attributes for a description/notes and priority should be added.
We will also have to think about how we want to save the state of the trackable ("backlog", "in progress", "finished", "aborted")
Potentially also release date (should be null in most cases since this would be only relevant for unreleased media)

Reply:
Was also thinking about saving trackable creation date for sorting. But since the ids are autoincrementing, sorting by ids achieves the same as sorting by creation date.